File: client/src/androidTest/java/com/jraska/github/client/TestRunner.java
package com.jraska.github.client;

import android.app.Application;
import android.content.Context;
import android.support.test.runner.AndroidJUnitRunner;

@SuppressWarnings("unused") // build.gradle
public final class TestRunner extends AndroidJUnitRunner {
  @Override
  public Application newApplication(ClassLoader cl, String className, Context context) throws InstantiationException, IllegalAccessException, ClassNotFoundException {
    return super.newApplication(cl, TestUITestApp.class.getName(), context);
  }
}

File: app/src/androidTest/java/com/jraska/github/client/TestRunner.kt
package com.jraska.github.client

import android.app.Application
import android.content.Context
import androidx.test.runner.AndroidJUnitRunner

@Suppress("unused") // build.gradle
class TestRunner : AndroidJUnitRunner() {
  override fun newApplication(cl: ClassLoader, className: String, context: Context): Application {
    return super.newApplication(cl, TestUITestApp::class.java.name, context)
  }
}
